manifests/repo.pp

```diff
@@ -30,63 +30,110 @@
   $username             = undef,
   $password             = undef,
   $description          = undef,
-  $environment          = $::environment,
+  $environment          = "${::environment}", # lint:ignore:only_variable_string
```

Comment:
sorry, I missed that. It might be connected to specific Puppet version, I'll check and come back to this soon.